src/oidcmsg/configure.py

```diff
@@ -105,12 +105,12 @@ def __getattr__(self, item, default=None):
             return default
 
     def __setattr__(self, key, value):
-        if key in self:
+        if key in self and self.key is not None:
```

Comment:
... or, consider if usefull, `self.key not in (None, "", {}, [] ...)`

self.key would only be None?